schemas/2016-09-01/Microsoft.Authorization.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2016-09-01/Microsoft.Authorization.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Authorization",
+  "description": "Microsoft Authorization Resource Types",
+  "resourceDefinitions": {
+    "locks": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Authorization/locks"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2016-09-01"
+          ]
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/ManagementLockProperties"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The properties of the lock."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the lock."
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties"
+      ],
+      "description": "Microsoft.Authorization/locks"
+    }
+  },
+  "definitions": {
+    "ManagementLockOwner": {
+      "type": "object",
+      "properties": {
+        "applicationId": {
+          "type": "string",
+          "description": "The application Id of the management lock owner."
+        }
+      }
+    },
+    "ManagementLockProperties": {
+      "type": "object",
+      "properties": {
+        "level": {
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [
+                "NotSpecified",
+                "CanNotDelete",
+                "ReadOnly"
+              ]
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The lock level of the management lock."
+        },
+        "notes": {
+          "type": "string",
+          "description": "The notes of the management lock."
+        },
+        "owners": {
```

Comment:
Yes please do that. We added the property for internal test. We need to generate SDK with the new property.